cli/cmd/install.go

```diff
@@ -252,6 +253,7 @@ func init() {
 	installCmd.Flags().StringSliceVar(&userInputIgnoredNamespaces, "ignore-namespace", k8sconsts.DefaultIgnoredNamespaces, "namespaces not to show in odigos ui")
 	installCmd.Flags().StringSliceVar(&userInputIgnoredContainers, "ignore-container", k8sconsts.DefaultIgnoredContainers, "container names to exclude from instrumentation (useful for sidecar container)")
 	installCmd.Flags().StringSliceVar(&userInputInstallProfiles, "profile", []string{}, "install preset profiles with a specific configuration")
+	installCmd.Flags().StringVarP(&uiMode, "ui-mode", "", "", "set the ui mode (one-of: readonly)")
```

Comment:
since odigos is already installed in their cluster, we will need to also allow updating this value via some cli config update command